c2corg_ui/static/partials/card_outings.html

```diff
@@ -0,0 +1,15 @@
+<a ng-click="cardCtrl.openCardPage()" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <div class="outing-author text-center">
+    <div class="outing-date">{{ doc['date_end'] | date: 'dd/MM/yyyy'}}</div>
```

Comment:
No clue :)
It's just a port from a Mako helper:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/helpers/view.html#L310-L313